app/modules/import-export/processes/import.php

```diff
@@ -413,6 +417,31 @@ public function init_import_session() {
 		update_option( Module::OPTION_KEY_ELEMENTOR_IMPORT_SESSIONS, $import_sessions, false );
 	}
 
+	/**
+	 * Get the manifest thumbnail, goes to the home page thumbnail if main doesn't exist
+	 *
+	 * @return string
+	 */
+	private function get_manifest_thumbnail(): string {
+		if ( ! empty( $this->manifest['thumbnail'] ) ) {
+			return $this->manifest['thumbnail'];
+		}
+
+		$response = wp_remote_get( static::API_KITS_URL );
+		if ( is_wp_error( $response ) || \WP_Http::OK !== wp_remote_retrieve_response_code( $response ) ) {
+			return '';
+		}
+
+		$kits = json_decode( wp_remote_retrieve_body( $response ) , true );
+		foreach ( $kits as $kit ) {
+			if ( $this->manifest['title'] === $kit['title'] ) {
+				return $kit['thumbnail'];
+			}
+		}
+
+		return '';
+	}
```

Comment:
Can't you get the individual kit instead of all kits?

Reply:
We don't have the API Kit ID in the manifest